pkg/namespace/namespace.go

```diff
@@ -0,0 +1,42 @@
+// Copyright © 2019 The Tekton Authors.
```

Comment:
Should this `// Copyright © 2019 The Tekton Authors.` be `// Copyright © 2020 The Tekton Authors.`